examples/flakes/flake.nix

```diff
@@ -8,13 +8,14 @@
 {
   description = "A basic nix-bitcoin node";
 
-  inputs.nix-bitcoin.url = "github:fort-nix/nix-bitcoin/release";
-  # You can also use a version branch to track a specific NixOS release
-  # inputs.nix-bitcoin.url = "github:fort-nix/nix-bitcoin/nixos-23.05";
+  inputs.nix-bitcoin.url = "github:fort-nix/nix-bitcoin/nixos-23.05";
```

Comment:
Doesn't this mean that `nix flake update` will not actually update your system anymore when we update to a new NixOS release? That seems undesirable - at least as a default option. Is the user supposed to notice that there are not updates coming from the nix-bitcoin inputs anymore?

Reply:
Agreed. I've reverted this change.